meteor/server/publications/timeline.ts

```diff
@@ -31,6 +32,16 @@ meteorPublish(PubSub.timeline, function (selector, token) {
 	}
 	return null
 })
+meteorPublish(PubSub.timelineDatastore, function (selector, token) {
```

Comment:
I think it's good to have this bundled, if you look at i.e. rundown.ts in the same folder you'll see that that has a lot of publications bundled as well. Since the datastore is so closely related to the timeline I think this makes sense.